test-util/src/main/java/io/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -455,6 +459,35 @@ private StringBuilder summarizeRejection(final Record<?> record) {
         .append(record.getRejectionReason());
   }
 
+  private String summarizeTimer(final Record<?> record) {
+    final var value = (TimerRecordValue) record.getValue();
+    final var builder = new StringBuilder();
+    final var dueTime = Instant.ofEpochMilli(value.getDueDate()).atZone(ZoneId.systemDefault());
+
+    builder.append("@").append(formatId(value.getTargetElementId()));
+
+    if (value.getElementInstanceKey() > 0) {
+      builder.append("[").append(shortenKey(value.getElementInstanceKey())).append("]");
+    } else {
+      builder.append(" (start event)");
+    }
```

Comment:
Following DRY, we should use `summarizeElementInformation()` instead. 
For timer start events, the element instance key could be `-1` or `?`.